veggieseasons/lib/widgets/search_bar.dart

```diff
@@ -39,6 +39,11 @@ class SearchBar extends StatelessWidget {
               child: CupertinoTextField(
                 controller: controller,
                 focusNode: focusNode,
+                decoration: BoxDecoration(
```

Comment:
nit: [`CupertinoTextField`'s documentation](https://master-api.flutter.dev/flutter/cupertino/CupertinoTextField-class.html) states that "If you set the decoration property to null, the decoration will be removed entirely".  Maybe we should just set it to `null`?

Reply:
@LongCatIsLooong I agree. I have set the decoration property to null. I was not sure how to do it so I just did `decoration: BoxDecoration(),`